samples/bluetooth/central_iso/prj.conf

```diff
@@ -1,11 +1,3 @@
 CONFIG_BT=y
 CONFIG_LOG=y
 CONFIG_BT_ISO_CENTRAL=y
-CONFIG_BT_SMP=y
```

Comment:
We could consider adding a note to the readme on how to encrypt data.

Reply:
Indeed. The sample uses
```c
#if defined(CONFIG_BT_SMP)
	iso_chan.required_sec_level = BT_SECURITY_L2,
#endif /* CONFIG_BT_SMP */
```
So that if SMP is enabled, it automatically encrypts the CIS